packages/docusaurus-theme-bootstrap/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "docusaurus-theme-bootstrap",
+  "version": "1.0.0",
```

Comment:
This version should be the same as our other packages

Reply:
Is there a reason for this? Or is just to keep the consistency?

Reply:
Because we are using [Lerna](https://lerna.js.org/) and our repo is a monorepo, so all packages should be the same version and we can use the latest (master) version of packages requiring them to be published.